### PR TITLE
Changes requested in the demo

### PR DESCRIPTION
**Pull request by isuruh15**

### CODE REVIEW

File: native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java [hunk 1]
```diff
@@ -277,11 +273,15 @@ public boolean engageSubCommand(List<String> argList) {
...
                             JsonElement overrideConfig = new Gson().toJsonTree(packageName);
                             toolConfigInstance.overrideConfig("packageConfig.name", overrideConfig);
-                        } else if (igName != null && !igName.isEmpty()) {
-                            JsonElement overrideConfig = new Gson().toJsonTree(igName);
-                            toolConfigInstance.overrideConfig("packageConfig.name.append", overrideConfig);
                         } else {
-                            JsonElement overrideConfig = new Gson().toJsonTree(HealthCmdUtils.getDirectories(specificationPath).get(0));
+                            String igName = "";
+                            //todo: For loop can be removed here hence tool optimized for single IG use case.
+                            for (Map.Entry<String, FHIRImplementationGuide> key :
```
Comment (sameeragunarathne) [inline native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java:279]:
```suggestion
                            for (Map.Entry<String, FHIRImplementationGuide> entry :
```

File: native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java [hunk 2]
```diff
@@ -277,11 +273,15 @@ public boolean engageSubCommand(List<String> argList) {
...
+                                    ((FHIRSpecificationData) fhirToolLib.getToolContext().getSpecificationData()).
+                                            getFhirImplementationGuides().entrySet()) {
+                                igName = key.getValue().getName();
+                            }
+                            JsonElement overrideConfig = new Gson().toJsonTree(igName);
```
Comment (sameeragunarathne) [inline native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java:284]: igName should be all-lowercased when using for the package name. is that logic added with this?

File: native/health-cli/src/main/resources/ballerina-health.help [hunk 3]
```diff
@@ -36,14 +42,11 @@ COMMANDS
...
                         which the command is run.
-                    --ig-name <name-of-the-implementation-guide>
-                        Name of the Implementation guide to be loaded to the tool. By default,
-                        name of the directory which contain FHIR specification files will be
-                        taken as the ig-name.
                     --package-name <fully-qualified-name-of-package>
                         Name of the Ballerina package to be generated.
                         Package name can be explicitly set using this argument. Unless specified,
-                        the --ig-name will be taken as the name of the package
+                        default name of the implementation guide will be taken as the
```
Comment (sameeragunarathne) [inline native/health-cli/src/main/resources/ballerina-health.help:48]:
```suggestion
                        default name of the implementation guide will be taken to construct the
```

File: native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java (hunk 2 above)
Comment (isuruh15) [reply native/health-cli/src/main/java/io/ballerina/health/cmd/fhir/FhirSubCmd.java]: fixed in 
ad188e1

**Review by sameeragunarathne — APPROVED** [review]

**Pull request merged**